QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
+QUEST_20150511_001985	$It would be nice if all these Revelators being here meant that the Goddesses are back.
+QUEST_20150511_001986	$Power is the most important attribute for swordsman. {nl}You could also train yourself to be agile and possess strong stamina. {nl}You should first think about what to become from now on.
+QUEST_20150511_001987	$Power is the most important attribute for archers. {nl}But, of course you can't ignore agile movement. {nl}You should first think about what to become from now on.
+QUEST_20150511_001988	$Intelligence is the most important attribute for Clerics. {nl}But sometimes you many need strong power or spirit. {nl}You should first think about what to become from now on. 
+QUEST_20150511_001989	$Helping reconstruct Fedimian may be more meaningful than pottery. {nl}But people need something to soothe their hearts.
+QUEST_20150511_001990	$If I had the diary, I could continue my adventure, but I wasn't selected… {nl}But if it were you…
+QUEST_20150511_001991	$If you cleverly use the dry air that forms around severe colds, you'd be able to freeze any object even in a desert. {nl}However, if you can't receive the blessing from the Goddess, it won't be possible to make that kind of air.
+QUEST_20150511_001992	$I'll report the damages. {nl}Chasing a Sparnas, a gigantic bird-type creature, will be hard. Despite being large, they are rather agile. {nl}It usually goes after the supplies, but I can't imagine nor find out why. I'm guessing someone's playing a trick.
+QUEST_20150511_001993	$Grain pocket was discovered in the ashes of the supplies. {nl}They smell so bad and old they can't be used for food at all. {nl}Please check the supplies for anything missing.
+QUEST_20150511_001994	$Defeat all the monsters nearby. {nl}The vicious energy in the vine forest is constantly creating new monsters. {nl}But you should first defeat most of them before you face Ironbaum. {nl}Even a small factor could ruin the big task. {nl}Also, you could try threatening the monsters so they can't do what they want to do.
+QUEST_20150511_001995	$(Everything is scribbled so poorly that they can't be distinguished from letters or drawings.)
+QUEST_20150511_001996	$The facility that belonged to the workers who worked on the Mausoleum of Zachariel suddenly shut down. {nl}It's not like they left because it was useless but they all just vanished all at once. {nl}Maybe it is related to the irregularly big size of the tomb and its complicated composition {nl}even after considering the fact that it was the grave to commemorate the greatest king in history.
+QUEST_20150511_001997	$King Zachariel had left many arrangements for future generations. {nl}We and the Jonas family were one of them. {nl}But the meaning of it is only known by the Goddess. {nl}At least we know that the will of the king that we are protecting and the secret of the tomb indicates one person.
```

Comment:
> King Zachariel had left many arrangements 

Wouldn't "King Zachariel had made many arrangements" be better?

> But the meaning of it is only known by the Goddess.

If this "it" refers to the arrangements, then it should be changed to something like:
"But their meaning is only known by the Godess."
Or, to make it less confusing:
"But the meaning of the arrangements is only known by the Godess."

>  At least we know that the will of the king that we are protecting and the secret of the tomb indicates one person.

The meaning of this line eludes me. Seems machine translated.